Cargo.toml

```diff
@@ -15,7 +15,7 @@ gpio_cdev = ["gpio-cdev"]
 default = [ "gpio_cdev", "gpio_sysfs" ]
 
 [dependencies]
-embedded-hal = { version = "0.2.3", features = ["unproven"] }
```

Comment:
should this reference the alpha crate version for now?

Reply:
This PR is based on the `1.0.0-alpha.1` PR #43. Further down in the file I patch the embedded-hal dependency to my branch (which sits on top of `1.0.0-alpha.1`).